_notifications-plugin/api.md

```diff
@@ -0,0 +1,401 @@
+ses_account.from_address | The Amazon SES account's sender email address.
+
+## Create channel configuration
+
+To create a notification channel configuration, send a POST request to the `configs` resource.
+
+#### Sample Request
+
+```json
+POST /_plugins/_notifications/configs/
+{
+  "config_id": "sample-id",
+  "name": "sample-name",
+  "config": {
+    "name": "Sample Slack Channel",
+    "description": "This is a Slack channel",
+    "config_type": "slack",
+    "is_enabled": true,
+    "slack": {
+      "url": "https://sample-slack-webhook"
+    }
+  }
+}
+```
+
+The create channel API operation accepts the following fields in its request body:
+
+Field |	Data Type |	Description |	Required
+:--- | :--- | :--- | :---
+config_id | String | The configuration's custom ID. | No
+config | Object |	Contains all relevant information, such as channel name, configuration type, and plugin source. |	Yes
+name | String |	Name of the channel. | Yes
+description |	String | The channel's description. | No
+config_type |	String | The destination of your notification. Valid options are `sns`, `slack`, `chime`, `webhook`, `smtp_account`, `ses_account`, `email_group`, and `email`. | Yes
+is_enabled | Boolean | Indicates whether the channel is enabled for sending and receiving notifications. Default is true.	| No
+
+The create channel operation accepts multiple `config_types` as possible notification destinations, so follow the format for your preferred `config_type`.
+
+```json
+"sns": {
+  "topic_arn": "<arn>",
+  "role_arn": "<arn>" //optional
+}
+"slack": {
+  "url": "https://sample-chime-webhoook"
+}
+"chime": {
+  "url": "https://sample-amazon-chime-webhoook"
+}
+"webhook": {
+      "url": "https://custom-webhook-test-url.com:8888/test-path?params1=value1&params2=value2"
+}
+"smtp_account": {
+  "host": "test-host.com",
+  "port": 123,
+  "method": "start_tls",
+  "from_address": "test@email.com"
+}
+"ses_account": {
+  "region": "us-east-1",
+  "role_arn": "arn:aws:iam::012345678912:role/NotificationsSESRole",
+  "from_address": "test@email.com"
+}
+"email_group": { //Email recipient group
+  "recipient_list": [
+    {
+      "recipient": "test-email1@test.com"
+    },
+    {
+      "recipient": "test-email2@test.com"
+    }
+  ]
+}
+"email": { //The channel that sends emails
+  "email_account_id": "<smtp or ses account config id>",
+  "recipient_list": [
+    {
+      "recipient": "custom.email@test.com"
+    }
+  ],
+  "email_group_id_list": []
+}
+```
+
+The following example demonstrates how to create a channel using email as a `config_type`:
+
+```json
+POST /_plugins/_notifications/configs/
+{
+  "id": "sample-email-id",
+  "name": "sample-name",
+  "config": {
+    "name": "Sample Email Channel",
+    "description": "Sample email description",
+    "config_type": "email",
+    "is_enabled" : true,
+    "email" : {
+    "email_account_id" : "<email_account_id>",
+    "recipient_list" : [
+      "sample@email.com"
+    ]
```

Comment:
I think the formatting for this is invalid, it should look like (similar to the example above):
```
  "recipient_list": [
    {
      "recipient": "sample@email.com"
    }
  ]
```